docs/tasks/logs.rst

```diff
@@ -0,0 +1,259 @@
+Segregate Controller And Datapath Logs
+--------------------------------------
+
+Kanister uses structured logging to ensure that its logs can be easily
+categorized, indexed and searched by downstream log aggregation software.
+
+By default, Kanister logs are output to the controller's ``stderr`` in JSON
+format. Generally, these logs can be categorized into *system logs* and
+*datapath logs*.
+
+System logs are logs emitted by the Kanister to track important controller
+events like interactions with the Kubernetes APIs, CRUD operations on
+blueprints and actionsets etc.
+
+Datapath logs, on the other hand, are logs emitted by data mover task pods
+created by Kanister. These logs are streamed to the Kanister controller before
+the task pods are terminated to ensure they are not lost inadvertently. Datapath
+log lines usually include the ``LogKind`` field, with its value set to
+``datapath``.
+
+The rest of this documentation provides instructions on how to segregate
+Kanister's system logs from datapath logs using Loki_ and Grafana_.
+
+To run the provided commands, access to a Kubernetes cluster using the
+``kubectl`` and ``helm`` command-line tools is required.
+
+Follow the instructions in the :ref:`helm` page to deploy Kanister on the
+cluster.
+
+Deployments Setup
+=================
+
+The commands and screenshots in this documentation are tested with the following
+software versions:
+
+* Loki 2.5.0
+* Grafana 8.5.3
+* Promtail 2.5.0
+
+.. image:: img/logs-kanister-loki-grafana.png
+
+Let's begin by installing Loki. Loki is a datastore optimized for holding log
+data. It indexes log data via streams made up of logs, where each stream is
+associated with an unique set of labels.
+
+.. code-block:: bash
+
+  helm repo add grafana https://grafana.github.io/helm-charts
+
+  helm repo update
+
+  helm -n loki install --create-namespace loki grafana/loki \
+    --set image.tag=2.5.0
+
+Confirm that the Loki StatefulSet is successfully rolled out:
+
+.. code-block:: bash
+
+  kubectl -n loki rollout status sts/loki
+
+.. code-block:: bash
+
+  statefulset rolling update complete 1 pods at revision loki-6d7b88b95c...
```

Comment:
can we have these two in the same code block?

Reply:
Got it - I removed them since they aren't necessary.